src/main/app/forms/validation/validators/isCountrySupported.ts

```diff
@@ -19,6 +19,8 @@ export class CheckCountryConstraint implements ValidatorConstraintInterface {
   async validate (value: any | string, args?: ValidationArguments): Promise<boolean> {
     if (value === undefined || value === null || value === '') {
       return true
+    } else if (value.toUpperCase().startsWith('IM')) {
```

Comment:
does the postcode lookup service have these?

Reply:
it returns with an error when searching for the couple I've tried

Reply:
then why add this?

Reply:
This constraint allows postcodes that are not found in the postcode lookup as it is not 100% reliable. We do know that all post codes that begin with IM are not allowed so returning false will show them an appropriate error message

Reply:
The implementation is not very readable. Can you extract 'IM' into collection that states what post codes are blacklisted. Variable name will hopefully give us more context.

Also use regular `if` instead of `else if` since previous statement returns early.